src/goto-instrument/contracts/dynamic-frames/dfcc_cfg_info.cpp

```diff
@@ -0,0 +1,894 @@
+/*******************************************************************\
+
+Module: Dynamic frame condition checking for function and loop contracts.
+
+Author: Qinheping Hu, qinhh@amazon.com
+Author: Remi Delmas delmasrd@amazon.com
+
+Date: March 2023
+
+\*******************************************************************/
+
+#include "dfcc_cfg_info.h"
+
+#include <util/c_types.h>
+#include <util/format_expr.h>
```

Comment:
Do we need this here?

Reply:
No, we use `from_expr` instead of format in this file. I removed it.